### DISCUSSION TITLE
Disappearing elements when changing to Select Tool in Firefox (using local copy)

### DISCUSSION THREAD

**Discussion by simon-r-white** [SVG-Edit/svgedit#528] [Q&A]
**Describe the bug**
I have extracted the svg-edit latest release and am running it locally through Firefox, specifically visiting the url:
````
file:///home/user/svgedit-3.0.0/editor/svg-editor.html
````
Elements seem to disappear if you change to the Select Tool. If you add multiple elements without changing tool, then change to the Select Tool, only the last drawn element disappears.

This does not happen when visiting the [live demo](https://unpkg.com/svgedit@3.0.1/editor/svg-editor.html).

**To Reproduce**
1. Draw a line using the Free Hand Tool
2. Do not click anywhere else on the canvas
3. Change to the Select Tool (either by using the keyboard shortcut ``V`` or clicking with the mouse)
4. Click on the canvas (not the element just added)
5. The element just drawn disappears

**SVG-Edit environment (IMPORTANT)**
- File for SVG-Edit: `svg-editor.html`
- Version: 3.0.0
- Protocol: file

**Desktop (please complete the following information):**
 - OS: Linux(Ubuntu)
 - Browser: firefox
 - Version: 62.0.3 (64-bit)

**Comment by brettz9** [comment]
Yes, if you update to 3.0.1 (as opposed to 3.0.0), this regression has been fixed. Closing but feel free to comment further.
